tests/auxil/bot_method_checks.py

```diff
@@ -566,13 +580,12 @@ async def check_defaults_handling(
         method: The shortcut/bot_method
         bot: The bot. May be a telegram.Bot or a telegram.ext.ExtBot. In the former case, all
             default values will be converted to None.
-        return_value: Optional. The return value of Bot._post that the method expects. Defaults to
-            None. get_file is automatically handled. If this is a `TelegramObject`, Bot._post will
-            return the `to_dict` representation of it.
         no_default_kwargs: Optional. A collection of keyword arguments that should not have default
             values. Defaults to an empty frozenset.
 
     """
+    guess_return_type_name(method)
```

Comment:
a variable can be assigned here and then we can use it in line 622?

Reply:
I think this is a missed debug print 🙈